pkg/cluster/service.go

```diff
@@ -0,0 +1,129 @@
+package cluster
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/redhat-developer/app-services-cli/pkg/api/kas"
+	"github.com/redhat-developer/app-services-cli/pkg/cluster/kafka"
+	"github.com/redhat-developer/app-services-cli/pkg/cluster/serviceregistry"
+	"github.com/redhat-developer/app-services-cli/pkg/kafka/kafkaerr"
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+	registryPkg "github.com/redhat-developer/app-services-cli/pkg/serviceregistry"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+)
+
+type Service interface {
+	ResourceExists(ctx context.Context, c *KubernetesCluster, namespace string, serviceName string, opts Options) error
+	CreateResource(ctx context.Context, c *KubernetesCluster, namespace string, serviceID string, opts Options) error
+}
+
+type Kafka struct{}
```

Comment:
@craicoverflow is expert for golang patterns, but.. I think purpose of declaring interface is that it can be used as generic dependency - implementations should be in separate files.

Mixing kafka and service registry in single cluster package breaks our main rule - ability to separate services and fast addition of new ones.